### PR TITLE
Keras2 models

### PR DESCRIPTION
**Pull request by xiaxin1998**
add keras2.models.Model and Sequential

### CODE REVIEW

**Comment by xiaxin1998** [comment]
@zhichao-li

File: pyzoo/zoo/pipeline/api/keras2/models.py [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+from zoo.pipeline.api.keras.engine import KerasNet
+from zoo.pipeline.api.utils import remove_batch
+from bigdl.util.common import to_list, callBigDlFunc
+
+if sys.version >= '3':
+    long = int
+    unicode = str
+
+
+class Sequential(KerasNet):
```
Comment (zhichao-li) [inline pyzoo/zoo/pipeline/api/keras2/models.py]: Sequential(k1.Sequential) ?

File: pyzoo/zoo/pipeline/api/keras2/models.py [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+        Create a Python Model base on the given java value
+        :param jvalue: Java object create by Py4j
+        :return: A Python Model
+        """
+        model = Sequential(jvalue=jvalue)
+        model.value = jvalue
+        return model
+
+
+class Model(KerasNet):
```
Comment (zhichao-li) [inline pyzoo/zoo/pipeline/api/keras2/models.py]: Model(k1.Model) ?

File: zoo/src/test/scala/com/intel/analytics/zoo/pipeline/api/keras2/layers/Conv1DSpec.scala [hunk 3]
```diff
@@ -15,25 +15,20 @@
  */
 
 package com.intel.analytics.zoo.pipeline.api.keras2.layers
-
 import com.intel.analytics.bigdl.nn.abstractnn.AbstractModule
 import com.intel.analytics.bigdl.tensor.Tensor
 import com.intel.analytics.bigdl.utils.Shape
 import com.intel.analytics.zoo.pipeline.api.keras.layers.{Keras2Test, KerasBaseSpec}
 import com.intel.analytics.zoo.pipeline.api.keras.models.Sequential
```
Comment (zhichao-li) [inline zoo/src/test/scala/com/intel/analytics/zoo/pipeline/api/keras2/layers/Conv1DSpec.scala]: change to `com.intel.analytics.zoo.pipeline.api.keras2.models.Sequential` ?

**Pull request closed without merging**